doc/source/whatsnew/v1.5.0.rst

```diff
@@ -21,6 +21,7 @@ Styler
 
   - New method :meth:`.Styler.to_string` for alternative customisable output methods (:issue:`44502`)
   - Added the ability to render ``border`` and ``border-{side}`` CSS properties in Excel (:issue:`42276`)
+  - Added a new method :meth:`.Styler.concat` which allows adding customised footer rows to visualise additional calculations on the data, e.g. totals and counts etc. (:issue:`43875`)
```

Comment:
IMO I think this could use a dedicated section. Seems like a pretty significant enhancement!

Reply:
I usually edit the Styler later, just keep a record of improvements overtime and then do a write up just before release.